### DISCUSSION TITLE
Copilot spinning in circles and not returning results

### DISCUSSION THREAD

**Discussion by yanxke** [community/community#34925] [Copilot Conversations]
Recently, my Copilot's icon has been constantly spinning in circles and eventually times out without giving any suggestions.  I think it might be because the connection is blocked by the firewall.  Is there a way to specify a proxy for Copilot?  It's been pretty unusable lately and I am a paid user.

**Comment by davecheney** [comment]
@yanxke please see https://github.com/community/community/discussions/29127

**Reply by clarkjenifer33** [reply]
Solution: 
1. make sure your IDE is up to date
2. If you're not properly signed in to GitHub CoPilot. Solution: make sure you're signed in to GitHub CoPilot
Regards: [tmwhatsap](https://mygbapps.com/tm-whatsapp-apk-download)

**Reply by yanxke** [reply]
Thank you @davecheney .  I've added the proxy settings and it's still spinning sometimes.  The main problem is that it's working intermittently, so it's hard to diagnose exactly what's wrong.

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by yanxke** [reply]
[INFO] [default] [2022-10-03T22:38:32.433Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2022-10-03T22:38:33.827Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-10-03T22:38:33.992Z] Cancelled by contextual filter
[INFO] [default] [2022-10-03T22:38:34.343Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2022-10-03T22:38:35.306Z] Cancelled after extractPrompt
[INFO] [default] [2022-10-03T22:38:35.820Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-03T22:38:35.942Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2022-10-03T22:38:38.648Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-10-03T22:38:38.973Z] Cancelled by contextual filter

**Reply by davecheney** [reply]
Does open copilot - cntl enter — work?

**Reply by yanxke** [reply]
No, it does not.  Here is another example of the latest logs:

[INFO] [default] [2022-10-04T00:56:18.731Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-04T00:56:21.264Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-04T00:56:36.615Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-04T00:57:11.713Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-04T00:58:02.741Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex

Ctrl Enter stuck at: Synthesizing 0/10 solutions

**Reply by davecheney** [reply]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by yanxke** [reply]
It is very possible, and hence the question on the proxy.  I've set the proxy settings already, but I'm not sure if it's taking effect as it's working intermittently.  My browser traffic works just fine through the proxy.

**Reply by davecheney** [reply]
I recommend joining the discussion on https://github.com/orgs/community/discussions/29127 for proxy related matters.
